src/test/cli/radosgw-admin/help.t

```diff
@@ -351,6 +352,11 @@
   
   <date> := "YYYY-MM-DD[ hh:mm:ss]"
   
+  Dedup throttle options:
+     --max-bucket-index-ops        specify max bucket-index requests per second allowed for an RGW during dedup, 0 means unlimited
+     --max-metadata-ops            specify max metadata requests per second allowed for an RGW during dedup, 0 means unlimited
+     --stat                        display dedup throttle setting
```

Comment:
Perhaps make it more clear if these settings apply:

* To each RGW daemon in isolation
* To each RGW service, divided by the count of RGW daemons
* To potentially multiple RGW services

Reply:
this is per deamon